pyppeteer/page.py

```diff
@@ -1013,6 +1013,14 @@ async def setBypassCSP(self, enabled: bool) -> None:
         await self._client.send('Page.setBypassCSP', {'enabled': enabled})
 
     async def emulateMedia(self, mediaType: str = None) -> None:
+        """Deprecated in favor of ``emulateMediaType()``.
+        Please, consider to use ``emulateMediaType()`` instead.
+        """
+        await self.emulateMediaType(mediaType)
+        raise DeprecationWarning('Deprecated. Use page.emulateMediaType(mediaType) instead. '
+                                 'This method is kept around as an alias for backwards compatibility.')
```

Comment:
We don't actually want to _raise_ an error, just warn about it (admittedly confusing as DeprecationWarning is an Exception. I consider it best practice to just return the result of the wrapped function (even though it doesn't matter in this case.
```suggestion
        warnings.warn('This method is kept for backwards compatibility, but may be removed in a future version. Use Page.emulateMediaType(mediaType) instead', DeprecationWarning)
        return await self.emulateMediaType(mediaType)
```

Reply:
using `warning`